viz_scripts/mode_specific_metrics.ipynb

```diff
@@ -51,186 +55,131 @@
    "execution_count": null,
-   "id": "every-stanley",
+   "id": "romance-green",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Energy Impact Calculation\n",
-    "scaffolding.unit_conversions(expanded_ct)"
+    "data_eb = expanded_ct.query(f\"mode_confirm == '{mode_of_interest}'\")"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "empty-intensity",
+   "id": "seeing-court",
    "metadata": {},
    "outputs": [],
    "source": [
-    "file_suffix = scaffolding.get_file_suffix(year, month, program)"
+    "quality_text = scaffolding.get_quality_text(expanded_ct, data_eb)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "demanding-picking",
+   "id": "noble-joint",
    "metadata": {},
    "source": [
-    "# Ebike Data Only"
+    "### Distribution of Replaced_mode attribute"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "romance-green",
-   "metadata": {},
+   "id": "liked-shade",
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
-    "data_eb = expanded_ct.query(\"Mode_confirm == 'Pilot ebike'\")"
+    "# Note: The portion of pilot ebikes in the pie chart below is only when the mode_confirm is different that ebike. Example: mode_confirm was Walk, replaced_mode ebike.\n",
+    "# mode_confrim was Car, drove alone, replaced_mode ebike.\n",
+    "labels_rm = expanded_ct['Replaced_mode'].value_counts(dropna=True).keys().tolist()\n",
+    "values_rm = expanded_ct['Replaced_mode'].value_counts(dropna=True).tolist()\n",
+    "plot_title=\"Number of trips for each replaced mode (selected by users)\\n%s\" % quality_text\n",
+    "file_name= 'ntrips_replaced_mode%s' % file_suffix\n",
+    "pie_chart_mode(plot_title,labels_rm,values_rm,file_name)\n",
+    "alt_text = store_alt_text_pie(pd.DataFrame(values_rm, labels_rm), file_name, plot_title)"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "seeing-court",
+   "cell_type": "markdown",
+   "id": "loaded-expert",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "quality_text = scaffolding.get_quality_text_ebike(expanded_ct, data_eb)"
+    "### Trips by purpose for specified mode"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "id": "respiratory-breach",
-   "metadata": {},
+   "metadata": {
+    "scrolled": false
+   },
    "outputs": [],
    "source": [
     "labels_tp = data_eb['Trip_purpose'].value_counts(dropna=True).keys().tolist()\n",
     "values_tp = data_eb['Trip_purpose'].value_counts(dropna=True).tolist()\n",
-    "plot_title=\"Number of trips for each purpose for eBike only\\n%s\" % quality_text\n",
-    "file_name= 'ntrips_ebike_purpose%s.png' % file_suffix\n",
-    "pie_chart_purpose(plot_title,labels_tp,values_tp,file_name)"
+    "plot_title=f\"Number of trips for each purpose for {mode_of_interest} only\\n{quality_text}\"\n",
+    "file_name= 'ntrips_mode_purpose%s' % file_suffix\n",
```

Comment:
This should put in the mode of interest, not just the "mode" because otherwise it is too easy to confuse with overall purpose. At the very least, we should at least have `mode_of_interest` instead of mode, but this is not that much harder to do.

```suggestion
    "file_name= f'ntrips_{mode_of_interest}_purpose{file_suffix}'\n",
```